### ISSUE TITLE
EPP is set, even though it's disabled

### ISSUE THREAD

**Issue by CasperH2O** [Valkirie/HandheldCompanion#644]
Observed while testing from a fresh install. Seeing this in the log:

```
HandheldCompanion Information: 0 : Windows executable devenv.exe now has the foreground
HandheldCompanion Information: 0 : User requested EPP AC: 33, DC: 50
HandheldCompanion Information: 0 : Windows executable devenv.exe now has the foreground
HandheldCompanion Information: 0 : User requested EPP AC: 33, DC: 50
HandheldCompanion Information: 0 : Windows executable devenv.exe now has the foreground
HandheldCompanion Information: 0 : User requested EPP AC: 33, DC: 50
HandheldCompanion Information: 0 : Windows executable devenv.exe now has the foreground
HandheldCompanion Information: 0 : User requested EPP AC: 33, DC: 50
HandheldCompanion Information: 0 : Windows executable devenv.exe now has the foreground
HandheldCompanion Information: 0 : User requested EPP AC: 33, DC: 50
HandheldCompanion Information: 0 : Windows executable HandheldCompanion.exe now has the foreground
HandheldCompanion Information: 0 : User requested EPP AC: 33, DC: 50
HandheldCompanion Information: 0 : Windows executable HandheldCompanion.exe now has the foreground
HandheldCompanion Information: 0 : User requested EPP AC: 33, DC: 50
HandheldCompanion Information: 0 : Windows executable firefox.exe now has the foreground
HandheldCompanion Information: 0 : User requested EPP AC: 33, DC: 50
HandheldCompanion Information: 0 : Windows executable devenv.exe now has the foreground
HandheldCompanion Information: 0 : User requested EPP AC: 33, DC: 50
HandheldCompanion Information: 0 : Windows executable firefox.exe now has the foreground
HandheldCompanion Information: 0 : User requested EPP AC: 33, DC: 50
HandheldCompanion Information: 0 : Windows executable devenv.exe now has the foreground
HandheldCompanion Information: 0 : User requested EPP AC: 33, DC: 50
```

Default profile has EPP disabled, no other profiles have EPP.

**Comment by CasperH2O** [comment]
Should be fixed with 8c25f0e.

**Comment by CasperH2O** [comment]
Fixed.
